### PR TITLE
sarkars/crop_and_resize

### PR DESCRIPTION
**Pull request by sayantan-nervana**
Note: UT is disabled

### CODE REVIEW

File: test/test_image_ops.cpp [hunk 1]
```diff
@@ -25,6 +25,48 @@ namespace ngraph_bridge {
 
 namespace testing {
 
+// Test op: CropAndResize
+// Disabled till a backend starts supporting it
+
+TEST(ImageOps, DISABLED_CropAndResize) {
```
Comment (sayantan-nervana) [inline test/test_image_ops.cpp]: UT is disabled

**Comment by sayantan-nervana** [comment]
CI failing as its checking out nGraph Version:  v0.25.1-rc.9
Will move to rc.10 in https://github.com/tensorflow/ngraph-bridge/pulls/320, which should fix teh build failures

File: test/test_image_ops.cpp [hunk 2]
```diff
@@ -25,6 +25,49 @@ namespace ngraph_bridge {
...
+  AssignInputValuesRandom(box_ind);
+
+  // [crop_height, crop_width]
+  Tensor crop_size(DT_INT32, TensorShape({6, 5}));
+  AssignInputValuesRandom(crop_size);
+
+  // TODO check with non-zero extrapolation value and method="nearest"
+  auto attr =
+      ops::CropAndResize::Attrs().ExtrapolationValue(0.0).Method("bilinear");
+  // attr.Method("bilinear");
```
Comment (kanvi-nervana) [inline test/test_image_ops.cpp]: stray comment?

Comment (sayantan-nervana) [reply test/test_image_ops.cpp]: yes. removing

Comment (sayantan-nervana) [inline test/test_image_ops.cpp]:
```suggestion
```

File: test/test_image_ops.cpp [hunk 3]
```diff
@@ -25,6 +25,49 @@ namespace ngraph_bridge {
 
 namespace testing {
 
+// Test op: CropAndResize
+// Disabled till a backend starts supporting it
```
Comment (kanvi-nervana) [inline test/test_image_ops.cpp]:
```suggestion
// Disabled till a backend starts supporting it
// Add more tests when backend starts supporting
```

**Pull request merged**